tests/test_primitives.py

```diff
@@ -94,111 +68,109 @@ def primitive_tester(env, objects, primitives, primitives_args):
 
         for primitive, args in zip(primitives, primitives_args):
             try:
-                execute_controller(controller.apply_ref(primitive, *args), env)
+                execute_controller(controller.apply_ref(primitive, *args, attempts=3), env)
```

Comment:
With the flaky annotation do we still need multiple attempts or can we just do 1 here?

Reply:
??? unresolved. please leave an answer. does it not work?